pheno_utils/basic_plots.py

```diff
@@ -288,3 +289,41 @@ def show_fundus(fname: str) -> None:
     ax.set_xticks([])
     ax.set_yticks([])
     ax.axis('off')
+
+# %% ../nbs/01_basic_plots.ipynb 22
+def plot_top_n_value_counts(df, column_name, n=20, filter_index=None, filter_value=None):
+    """
+    Plot the top N items in a specified column of a DataFrame.
+    
+    Parameters:
+    df (DataFrame): The DataFrame to analyze.
+    column_name (str): The name of the column to count and plot.
+    n (int): The number of top items to plot.
+    filter_index (str, optional): The name of the index to filter on, if any.
+    filter_value (str, optional): The value to filter the index on, if any.
+    """
+    
+    # If filter parameters are provided, filter the DataFrame
+    if filter_index and filter_value:
```

Comment:
I don't know if you considered this, but I sometimes find the .query() method very convenient for this sort of filtering, and exposing it to the user (e.g., via a `query_str` argument that is None by default), allows the user to filter by any combination of columns and indices. If you still prefer to user filter_index/value that's fine.